FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/PlottingMethod.java

```diff
@@ -0,0 +1,14 @@
+package hec.ensemble.stats;
+
+public class PlottingMethod {
```

Comment:
this seems ok. does it make sense to have plotting method have a default behavior that is overriden rather than an abstract behavior that is implemented in the children? consider making "method" abstract. Also, method is actually computing probabilities, so maybe have it named more appropriately.  like "computeProbabilities".
It seems that the values are only used for getting the size of the array and to determine n+1 for weibull, do you think the values array will be used? if not, why not just accept an int "number of odrinates" argument

Reply:
Changed to interface and renamed the method to computeProbability.  Also added Median for some variety and testing different plotting cases.

The values are used in the PlottingPositionComputable class orderValues() and assignProbability() methods.  I needed a way to assign the probability to each value for plotting it later.